### DISCUSSION TITLE
Which package to install on openSUSE Tumbleweed?

### DISCUSSION THREAD

**Discussion by matteo-gambarutti** [trapexit/mergerfs#1351] [Q&A]
Hi,
I would like to install mergerfs on openSUSE Tumbleweed and I see there are RPMs which Zypper support.
I can't understand if I have to choose from el or fc and which number (el9, fc40...).

Also, I tried to compile from source: during the "make" step, a lot of warnings showed up (unused vars, vars type etc...). Is this normal?

Thank you.

**Comment by trapexit** [comment]
Those rpms are for RHEL (el) and Fedora (fc). I don't build openSUSE RPMs. I could look into it but nothing there today. 

You could use the static builds too.

Yes, it is normal. If it compiles it likely works.

**Reply by trapexit** [reply]
You can also just try to build the RPM.

**Reply by matteo-gambarutti** [reply]
Thank for your reply!

Building for openSUSE: not sure how popular it is and if is something you want to do. Up to you...
Static builds: are builds that work for any Linux distribution as all the needed objects are present (linked) at the compile time?
I asked about the warnings as I still have nightmare from delivering a small project during my studies and one point was NO WARNINGS AT COMPILE TIME :) 

Building the RPM: how is it done?

**Reply by trapexit** [reply]
If they have a container image that allows me to add it to my build pipeline I can. I've not used SuSE in over 20 years so I'm a bit out of the loop of what they do.

Yes, the static build is built statically meaning all dependencies are compiled in.

It's in the build section of the docs: https://github.com/trapexit/mergerfs?tab=readme-ov-file#rhel--centos--rocky--fedora

**Comment by jeffgt14** [comment]
If you don't mind third party repos, the filesystems repo is good at staying updated.

https://build.opensuse.org/package/show/filesystems/mergerfs

**Reply by matteo-gambarutti** [reply]
Thank you so much for this!
I don't mind third party repos but I have just one simple question as I'm new to openSUSE and extra repos: how do you trust a third party repository? How do you know if it's save to add a repo or not?

**Reply by jeffgt14** [reply]
Short answer is there's no 100% certain way. I just try to research them and put them at the lowest priority.
